Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -609,3 +617,169 @@ theorem val_le_one_iff_eq_coe (f : LaurentSeries K) : Valued.v f ≤ (1 : ℤₘ
 end LaurentSeries
 
 end AdicValuation
+namespace LaurentSeries
+section Complete
+
+open Filter
+
+open scoped DiscreteValuation
+
+variable {K : Type*} [Field K]
+
+/- Sending a Laurent series to its `d`-th coefficient is uniformly continuous when the coefficient
+field has the discrete uniformity. -/
+theorem uniformContinuous_coeff {uK : UniformSpace K} (h : uK = ⊥) (d : ℤ) :
```

Comment:
I have the (mathematical) impression that the theorem does not really need the hypothesis that the coefficient has the discrete uniformity.